src/components/container/ReasonCard/ReasonCard.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./ReasonCard.scss";
+
+function ReasonCard({ icon, title, description }) {
+  return (
+    <div className="ReasonCard">
+      <div className="ReasonCard__icon">
+        <div className="ReasonCard__icon--background">
+          <img className="ReasonCard__icon--image" src={icon.path} alt={icon.description} />
```

Comment:
@jasper-prgh Do `background` and `image` count as modifiers, or should they be subelements instead? e.g. `ReasonCard__icon__background`

Reply:
Didn't notice this. Yes, they should be elements instead

Reply:
should the image className then be 'ReasonCard__icon__background__image' or can i use 'ReasonCard__icon__image' to keep it short?

Reply:
actually, icon__background__image would be best imo. This way it might be able to be reused by other components as well :)